### PR TITLE
RA location management

### PR DESCRIPTION
**Pull request by relaxnow**

### CODE REVIEW

**Comment by relaxnow** [comment]
@rjkip could you review?

File: app/Resources/translations/messages.en_GB.xliff [hunk 1]
```diff
@@ -284,6 +294,31 @@
...
+      </trans-unit>
+      <trans-unit id="74e5be0ff6cd580084482794edee319576a8c961" resname="ra.form.ra_create_ra_location.label.contact_information">
+        <jms:reference-file line="37">Form/Type/CreateRaLocationType.php</jms:reference-file>
+        <source>ra.form.ra_create_ra_location.label.contact_information</source>
+        <target>Contact Information</target>
+      </trans-unit>
+      <trans-unit id="8a72daea02c0ad7fd2b4a8182c961f58ffda08e6" resname="ra.form.ra_create_ra_location.label.create_ra_location">
+        <jms:reference-file line="40">Form/Type/CreateRaLocationType.php</jms:reference-file>
+        <source>ra.form.ra_create_ra_location.label.create_ra_location</source>
+        <target>Create RA Location</target>
```
Comment (rjkip) [inline app/Resources/translations/messages.en_GB.xliff]: Small `L`

Comment (rjkip) [inline app/Resources/translations/messages.en_GB.xliff]: Nvm

File: app/Resources/translations/messages.nl_NL.xliff [hunk 2]
```diff
@@ -147,6 +147,16 @@
         <source>ra.auditlog.when</source>
         <target>Datum</target>
       </trans-unit>
+      <trans-unit id="dae6a1cc0ab3be03b10145c4af4996c87111e961" resname="ra.create_ra_location.changed">
+        <jms:reference-file line="151">RaBundle/Controller/RaLocationController.php</jms:reference-file>
+        <source>ra.create_ra_location.changed</source>
+        <target>RA Locatie succesvol bijgewerkt</target>
```
Comment (rjkip) [inline app/Resources/translations/messages.nl_NL.xliff]: Nitpick: koppelstreepje :wink:

File: app/Resources/translations/validators.en_GB.xliff [hunk 3]
```diff
@@ -190,6 +190,46 @@
         <source>middleware_client.dto.ra_listing.role.must_not_be_blank</source>
         <target>RA Listing Institution must be set</target>
       </trans-unit>
+      <trans-unit id="19651dd2288d557c78a5730d58c3bddc9c6d50e2" resname="middleware_client.dto.ra_location.contact_information.must_be_string">
+        <source>middleware_client.dto.ra_location.contact_information.must_be_string</source>
+        <target state="new">middleware_client.dto.ra_location.contact_information.must_be_string</target>
```
Comment (rjkip) [inline app/Resources/translations/validators.en_GB.xliff]: Forgotten, or? :wink:

File: src/Surfnet/StepupRa/RaBundle/Command/ChangeRaLocationCommand.php [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    /**
+     * @Assert\NotBlank(message="ra.accredit_candidate.location.may_not_be_blank")
+     * @Assert\Type(type="string")
+     *
+     * @var string
+     */
+    public $location;
+
+    /**
+     * @Assert\NotBlank(message="ra.accredit_candidate.contact_information.may_not_be_blank")
```
Comment (rjkip) [inline src/Surfnet/StepupRa/RaBundle/Command/ChangeRaLocationCommand.php]: Afaik contact information *may* be an empty string.

File: src/Surfnet/StepupRa/RaBundle/Command/ChangeRaLocationCommand.php [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+    /**
+     * @var string
+     */
+    public $institution;
+
+    /**
+     * @var string
+     */
+    public $currentUserId;
```
Comment (rjkip) [inline src/Surfnet/StepupRa/RaBundle/Command/ChangeRaLocationCommand.php]: What do we need this for?

File: src/Surfnet/StepupRa/RaBundle/Command/ChangeRaLocationCommand.php [hunk 6]
```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * Copyright 2014 SURFnet bv
```
Comment (rjkip) [inline src/Surfnet/StepupRa/RaBundle/Command/ChangeRaLocationCommand.php]: `$year->update()` :x

File: src/Surfnet/StepupRa/RaBundle/Resources/views/RaLocation/manage.html.twig [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+                                    {{ 'ra.ra_location.manage.edit'|trans }}
+                                </a>
+                                <button class="btn btn-warning remove"
+                                        data-toggle="modal"
+                                        data-target="#removalModal"
+                                        data-sfid="{{ location.id }}"
+                                        data-sfinstitution="{{ location.institution }}"
+                                        data-sfname="{{ location.name }}"
+                                        data-sflocation="{{ location.location }}"
+                                        data-sfcontactInformation="{{ location.contactInformation }}"
```
Comment (rjkip) [inline src/Surfnet/StepupRa/RaBundle/Resources/views/RaLocation/manage.html.twig]: Nit: these `sf`s stand for second factor I presume.

**Comment by rjkip** [comment]
Nearly there, some nits :)

File: src/Surfnet/StepupRa/RaBundle/Command/ChangeRaLocationCommand.php (hunk 4 above)
Comment (relaxnow) [inline src/Surfnet/StepupRa/RaBundle/Command/ChangeRaLocationCommand.php]: True, but we have to wrap this up now, so we'll make it obligatory for now.

File: src/Surfnet/StepupRa/RaBundle/Command/ChangeRaLocationCommand.php (hunk 5 above)
Comment (relaxnow) [inline src/Surfnet/StepupRa/RaBundle/Command/ChangeRaLocationCommand.php]: Logging if a ChangeRaLocationCommand failed

**Pull request merged**